core/rpc/src/chain/mod.rs

```diff
@@ -57,9 +57,9 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 	#[rpc(name = "chain_getBlockHash", alias("chain_getHead"))]
 	fn block_hash(&self, hash: Option<number::NumberOrHex<Number>>) -> Result<Option<Hash>>;
 
-	/// Get hash of the last finalised block in the canon chain.
-	#[rpc(name = "chain_getFinalisedHead")]
-	fn finalised_head(&self) -> Result<Hash>;
+	/// Get hash of the last finalized block in the canon chain.
+	#[rpc(name = "chain_getFinalizedHead")]
```

Comment:
Can we create an alias for the old names? From a middleware perspective there is no way to support both types of chains. So really from a use perspective in the API and UIs this rename is quite problematic without an alias for the old - there needs to be time to swap over.

Reply:
Not sure if aliases are possible. If not I'd suggest we just bite the bullet and only support post 1.0 from here on.

Reply:
aliases are supported via `#[rpc(name = ".", alias("a"), alias("b")]`